package.json

```diff
@@ -52,6 +52,7 @@
     "react-addons-test-utils": "~15.1.0",
     "react-bootstrap": "0.29.3",
     "react-cropper": "^0.9.4",
+    "react-datepicker": "^0.29.0",
```

Comment:
no changes to `npm-shrinkwrap.json`? might have to:

```sh
git checkout master -- package.json npm-shrinkwrap.json
# reboot docker-compose containers
docker-compose run watch npm prune
docker-compose run watch npm install --save react-datepicker
```

`npm` is just such a fun piece of software...